seaworthy/tests-core/test_dockerhelper.py

```diff
@@ -9,12 +9,12 @@
 # We use this image to test with because it is a small (~7MB) image from
 # https://github.com/docker-library/official-images that runs indefinitely with
 # no configuration.
-TEST_IMAGE = 'nginx:alpine'
+IMG = 'nginx:alpine'
```

Comment:
Were you remembering the Gestalt demo? 😛

Reply:
I wish I had. I started with `nginx:latest` and then discovered the hard way that it's quite a bit bigger...